libbeat/docs/yaml.asciidoc

```diff
@@ -94,3 +94,19 @@ to an integer. If not, it's converted to a float.
 To prevent unwanted type conversions, avoid using leading zeros in field values,
 or wrap the values in single quotation marks.
 
+[float]
+[[dollar-sign-strings]]
+=== Avoid accidental template variable resolution
+
+The templating engine that allows the config to resolve data from environment 
+variables can result in errors in strings with `$` characters. For example, if a 
+password field contains `$$`, the engine will resolve this to `$`. To work around this,
```

Comment:
```suggestion
password field contains `$$`, the engine will resolve this to `$`.

To work around this,
```

Reply:
OK, this is really weird, but without the paragraph break, our doc build can't handle the double $$ signs and the conditional coding in the same paragraph. You end up with:

![image](https://user-images.githubusercontent.com/14206422/78825611-240ad880-7995-11ea-9bb4-a0960ff8517c.png)

I think this is probably a bug in how our doc build handles conditional coding (I know Nik E made some hacks to make our coding work in asciidoctor). Anyhow, adding the empty line is an easy fix. Not sure we want to dig into why this is happening.